harmonica/tests/test_point_gravity.py

```diff
@@ -89,237 +90,105 @@ def test_invalid_masses_array():
 # ---------------------------
 # Cartesian coordinates tests
 # ---------------------------
-@pytest.mark.use_numba
-def test_potential_cartesian_known_values():
+
+
+@pytest.fixture(name="point_mass")
+def fixture_point_mass():
     """
-    Compare the computed gravitational potential with reference values
+    Defines a point located in the origin with a mass of 500kg
     """
-    # Define a single point mass
-    point_mass = [0, 0, 0]
+    point = [0, 0, 0]
     mass = [5000]
-    # Define a set of computation points
+    return point, mass
+
+
+@pytest.fixture(name="coordinates")
+def fixture_coordinates():
+    """
+    Define a set of sample computation points
+    """
+    # fmt: off
     northing = np.array(
-        [
-            931.10,
-            -91.03,
-            -849.97,
-            365.09,
-            -276.50,
-            232.36,
-            312.70,
-            -134.88,
-            409.50,
-            -346.05,
-            557.02,
-            -932.30,
-            -300.25,
-            -110.90,
-            -583.77,
-            -546.05,
-            -95.70,
-            -734.09,
-            290.47,
-            993.40,
-            684.20,
-            437.78,
-            -677.84,
-            -304.01,
-            -807.34,
-            -376.36,
-            900.98,
-            -955.56,
-            576.78,
-            223.98,
-            -13.06,
-            830.86,
-            339.04,
-            -285.78,
-            366.23,
-            174.36,
-            48.05,
-            820.27,
-            -108.76,
-            -38.23,
-            -484.50,
-            312.16,
-            -380.09,
-            -334.75,
-            77.81,
-            902.84,
-            847.66,
-            450.64,
-            791.34,
-            622.71,
-        ]
+        [931.10, -91.03, -849.97, 365.09, -276.50, 232.36, 312.70, -134.88,
+         409.50, -346.05, 557.02, -932.30, -300.25, -110.90, -583.77, -546.05,
+         -95.70, -734.09, 290.47, 993.40, 684.20, 437.78, -677.84, -304.01,
+         -807.34, -376.36, 900.98, -955.56, 576.78, 223.98, -13.06, 830.86,
+         339.04, -285.78, 366.23, 174.36, 48.05, 820.27, -108.76, -38.23,
+         -484.50, 312.16, -380.09, -334.75, 77.81, 902.84, 847.66,
+         450.64, 791.34, 622.71]
```

Comment:
The `fmt: off` comment is there to prevent Black from reformating the arrays: it would put a single element per line.